### PR TITLE
add issue template so people can be directed to the forums naturally

### PR DESCRIPTION
**Pull request by danbills**
Not sure if this Markdown is going to render properly as there is no preview.  May take a couple attempts to get this right

### CODE REVIEW

**Comment by Horneth** [comment]
IntelliJ has a Markdown plugin that gives you a preview FWIW

**Comment by danbills** [comment]
So it occurs to me now that this is hidden from markdown output on purpose!  (It's a comment).

Thus I will just paste URLs.

File: .github/issue_template.md [hunk 1]
```diff
@@ -0,0 +1,19 @@
+<!--
+Hi!  Thanks for taking the time to report feedback.
+
+Are you concerned with running your own instance of Cromwell,e.g. config issues?  Then please report your issue to the WDL Forum:
```
Comment (cjllanwarne) [inline .github/issue_template.md]: Space needed after `Cromwell,`

File: .github/issue_template.md [hunk 2]
```diff
@@ -0,0 +1,19 @@
...
+https://gatkforums.broadinstitute.org/wdl/categories/ask-the-wdl-team
```
Comment (cjllanwarne) [inline .github/issue_template.md]: Ideally we'd be able to say:
* WDL question -> WDL forum
* CWL question -> CWL gitter/forum
* Config question -> read the docs/Cromwell forum
* Firecloud question -> FireCloud forum
* Bug: fill in this template...

File: .github/issue_template.md [hunk 3]
```diff
@@ -0,0 +1,19 @@
...
+
+Are you having an issue w/ Firecloud?  Then please report your issue to the Firecloud Forum:
```
Comment (mcovarr) [inline .github/issue_template.md]: w/ => with

Comment (mcovarr) [reply .github/issue_template.md]: FireCloud not Firecloud

File: .github/issue_template.md [hunk 4]
```diff
@@ -0,0 +1,19 @@
...
+
+Are you seeing something that looks like a bug?  Then great!  You're in the right place.
+-->
+
+
+<!-- Which backend are you running?  -->
+
+<!-- Paste/Attach your workflow if possible: -->
+
+<!-- Paste your configuration if possible: -->
```
Comment (mcovarr) [inline .github/issue_template.md]: perhaps with a friendly reminder to redact any sensitive information

File: .github/issue_template.md (hunk 3 above)
Comment (mcovarr) [reply .github/issue_template.md]: Forum => forum

File: .github/issue_template.md (hunk 4 above)
Comment (cjllanwarne) [reply .github/issue_template.md]: Yeah actually this is super important. There's a lot of sensitive info in there and we 100% don't want people's mysql passwords or google tokens ending up in github...

File: .github/issue_template.md (hunk 1 above)
[the remark this answers is not in the document]
Comment (danbills) [reply .github/issue_template.md]: There are two spaces after `?`, not sure what you are suggesting?

File: .github/issue_template.md (hunk 2 above)
[the remark this answers is not in the document]
Comment (danbills) [reply .github/issue_template.md]: While I agree, this isn't markdown, it's a markdown comment.  thus it doesn't show up when one submits the issue.

I'm going to call it the Cromwell forum and link to WDL forum until we have them sorted out.

**Pull request merged**